ncs/app_envelope_encrypted.yaml.jinja2

```diff
@@ -0,0 +1,153 @@
+    SuitDigest:
+      suit-digest-algorithm-id: cose-alg-sha-256
+  suit-manifest:
+    suit-manifest-version: 1
+    suit-manifest-sequence-number: {{ sequence_number }}
+    suit-common:
+      suit-components:
+      - - MEM
+        - {{ application['dt'].label2node['cpu'].unit_addr }}
+        - {{ get_absolute_address(application['dt'].chosen_nodes['zephyr,code-partition']) }}
+        - {{ application['dt'].chosen_nodes['zephyr,code-partition'].regs[0].size }}
+      - - CAND_IMG
+        - 0
+      suit-shared-sequence:
+      - suit-directive-set-component-index: 0
+      - suit-directive-override-parameters:
+          suit-parameter-vendor-identifier:
+            RFC4122_UUID: {{ mpi_application_vendor_name }}
+          suit-parameter-class-identifier:
+            RFC4122_UUID:
+              namespace: {{ mpi_application_vendor_name }}
+              name: {{ mpi_application_class_name }}
+          suit-parameter-image-digest:
+            suit-digest-algorithm-id: cose-alg-sha-256
+            suit-digest-bytes:
+              file_direct: {{ application['encryption_artifacts_dir'] }}/plain_text_digest.bin
+          suit-parameter-image-size:
+            file_direct: {{ application['encryption_artifacts_dir'] }}/plain_text_size.txt
+      - suit-condition-vendor-identifier:
+        - suit-send-record-success
+        - suit-send-record-failure
+        - suit-send-sysinfo-success
+        - suit-send-sysinfo-failure
+      - suit-condition-class-identifier:
+        - suit-send-record-success
+        - suit-send-record-failure
+        - suit-send-sysinfo-success
+        - suit-send-sysinfo-failure
+      - suit-directive-set-component-index: 1
+      - suit-directive-override-parameters:
+          suit-parameter-image-digest:
+            suit-digest-algorithm-id: cose-alg-sha-256
+            suit-digest-bytes:
+              file: {{ application['encryption_artifacts_dir'] }}/encrypted_content.bin
+    suit-validate:
+    - suit-directive-set-component-index: 0
+    # In the case of streaming operations it is worth to retry them at least once.
+    # This increases the robustness against bit flips on the transport,
+    # for example when storing the data on an external memory device.
+    # The suit-directive-try-each will complete on the first successful subsequence.
+    - suit-directive-try-each:
+      - - suit-condition-image-match:
+          - suit-send-record-success
+          - suit-send-record-failure
+          - suit-send-sysinfo-success
+          - suit-send-sysinfo-failure
+      - - suit-condition-image-match:
+          - suit-send-record-success
+          - suit-send-record-failure
+          - suit-send-sysinfo-success
+          - suit-send-sysinfo-failure
+    suit-invoke:
+    - suit-directive-set-component-index: 0
+    - suit-directive-invoke:
+      - suit-send-record-failure
+    suit-install:
+    - suit-directive-set-component-index: 1
+    - suit-directive-override-parameters:
```

Comment:
This values are already assigned in shared-sequence so probably lines 73-78 are obsolete. I will double check during tests.

Reply:
Manifests reworked.